src/utils/DockerMachineUtil.js

```diff
@@ -60,31 +58,29 @@ var DockerMachine = {
       return false;
     });
   },
-  create: function (machineName = this.name(), provider) {
+  create: function (machineName = this.name(), provider = "hyperv", virtualBoxIsInstalled = true) {
```

Comment:
imo there is no need for the extra check of `virtualboxinstalled`. create is only called, if we are already sure, that we can call it.